src/globalStyles.tsx

```diff
@@ -1,13 +1,63 @@
-import { createGlobalStyle } from "styled-components"
+import { createGlobalStyle } from 'styled-components';
 
 const GlobalStyles = createGlobalStyle`
-  body {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-    width: 100vw;
-    background-color: #141414;
-  }
+  /* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
Nie jest to zła praktyka, ale teraz najczęściej instaluje się https://www.npmjs.com/package/normalize.css?activeTab=versions jako zależność projektu, importuje i gotowe :P